parent/pom.xml

```diff
@@ -634,8 +634,8 @@
     <splunk-version>1.6.4.0_1</splunk-version>
     <spring-batch-version>4.0.1.RELEASE</spring-batch-version>
     <spring-batch-bundle-version>4.0.1.RELEASE_1</spring-batch-bundle-version>
-    <spring-boot-version>2.0.6.RELEASE</spring-boot-version>
-    <spring-cloud-commons-version>2.0.2.RELEASE</spring-cloud-commons-version>
+    <spring-boot-version>2.1.0.RELEASE</spring-boot-version>
+    <spring-cloud-commons-version>2.0.1.RELEASE</spring-cloud-commons-version>
```

Comment:
I have not tried both versions. it seems spring-cloud-commons-version is back to 2.0.1.RELEASE? anything specific...

Reply:
Well spotted, my mistake, I'll update this